.github/workflows/qa.yml

```diff
@@ -31,17 +31,12 @@ jobs:
       - uses: actions/checkout@v2
       - uses: thebritican/fetch-latest-release@v2.0.0
         id: latest_release
-      - uses: actions/checkout@v2
-        with:
-          repository: SonarSource/sonar-scanning-examples
-          path: target_repository
       - name: Build image
```

Comment:
is there anything in the QA workflow that is relevant? Can't we remove it all?

Reply:
What about the `Generate CycloneDX SBOM`?